framework/src/materials/PiecewiseLinearInterpolationMaterial.C

```diff
@@ -47,7 +47,7 @@ PiecewiseLinearInterpolationMaterial::PiecewiseLinearInterpolationMaterial(
     _scale_factor(getParam<Real>("scale_factor")),
     _extrap(getParam<bool>("extrapolation")),
     _property(declareProperty<Real>(_prop_name)),
-    _dproperty(declarePropertyDerivative<Real>(_prop_name, getVar("variable", 0)->name()))
+    _dproperty(declarePropertyDerivative<Real>(_prop_name, coupledName("variable", 0)))
```

Comment:
So for constant coupling this would declare something like `dprop/d45.7`?

Reply:
the pattern should probalbly be 
```suggestion
    _dproperty(isCoupledConstant("variable") ? nullptr : &declarePropertyDerivative<Real>(_prop_name, coupledName("variable", 0)))
```

and changing `_dproperty` to a pointer